### PR TITLE
Bump dependencies

### PR DESCRIPTION
**Pull request by nikku**
House keeping + updating our properties panel foundations.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,6 +14,7 @@
     "format": "run-s format:markdown 'lint -- --fix'",
     "format:markdown": "remark . -qo",
     "test:distro": "node tasks/test-distro.mjs",
+    "start": "npm run start:cloud",
```
Comment (barmac) [inline package.json:17]: I am absolutely delighted with this change.

**Comment by barmac** [comment]
I noticed that there's something wrong with undo/redo in this:

https://github.com/camunda/camunda-bpmn-js/assets/28307541/643bd06f-18d8-442e-a999-079e7a7a3e79

**Review by barmac — COMMENTED** [review]
I noticed that the undo/redo interaction is broken both in FEEL popup as well as in the FEEL field. Do we want to ship it anyway right now and fix the bugs later, or rather bump deps in this project to a non-buggy distribution of the properties panel?

**Comment by barmac** [comment]
We can merge this and fix https://github.com/bpmn-io/bpmn-js-properties-panel/issues/980 as a follow-up if you will.

**Comment by nikku** [comment]
Let's get this merged :slightly_smiling_face:

**Pull request merged**